MiniApp/Classes/API/ListingAPI.swift

```diff
@@ -5,17 +5,26 @@ internal class ListingApi {
         self.environment = environment
     }
 
-    func createURLRequest() -> URLRequest? {
-        guard let url = getListingURL() else {
+    func createURLRequest(miniAppID: String? = nil) -> URLRequest? {
```

Comment:
`Optional`?

Reply:
yes, as it is in the API call. If I have the parameter I add it to the call to get the listing with only 1 mini app, else I make the regular call for the whole listing